### PR TITLE
remove non-free leanpub books

### PR DESCRIPTION
**Pull request by eshellman**
~~45~~ 40 books removed, the ~~59~~ 64 leanpub books remaining are still free.

Addresses #6852 

Remaining books either have free-to-read html, or they have price set to $0

### CODE REVIEW

File: books/free-programming-books-id.md [hunk 1]
```diff
@@ -169,22 +170,6 @@
...
-
-#### Angular
-
-* [Tutorial Angular Indonesia](https://degananda.com/tutorial-angular-indonesia-daftar-isi-download-pdf/) - Degananda Ferdian Priyambada (HTML)
-* [Tutorial Series Belajar Angular 2019](https://www.bewoksatukosong.com/2019/09/tutorial-series-belajar-angular-2019.html) - Bewok Satu Kosong (HTML)
-
-
-#### Deno
-
-* [Belajar Dengan Jenius Deno](https://raw.githubusercontent.com/gungunfebrianza/Belajar-Dengan-Jenius-DenoTheWKWKLand/master/Belajar%20Dengan%20Jenius%20Deno.pdf) - Gun Gun Febrianza (PDF)
```
Comment (davorpa) [inline books/free-programming-books-id.md:185]: Masking a resources movement. Maybe better leave as

```suggestion
### TypeScript

#### Angular

* [Tutorial Angular Indonesia](https://degananda.com/tutorial-angular-indonesia-daftar-isi-download-pdf/) - Degananda Ferdian Priyambada (HTML)
* [Tutorial Series Belajar Angular 2019](https://www.bewoksatukosong.com/2019/09/tutorial-series-belajar-angular-2019.html) - Bewok Satu Kosong (HTML)


#### Deno

* [Belajar Dengan Jenius Deno](https://raw.githubusercontent.com/gungunfebrianza/Belajar-Dengan-Jenius-DenoTheWKWKLand/master/Belajar%20Dengan%20Jenius%20Deno.pdf) - Gun Gun Febrianza (PDF)
```

**Review by davorpa — COMMENTED** [review]
Also there are some with `0$` that need membership after added to cart. E.g.:

- **`free-programming-books-es.md`**
    ```markdown
    * [JavaScript, ¡Inspírate!](https://leanpub.com/javascript-inspirate) (Leanpub cuenta requerida)
    ```

Could you review again?

File: books/free-programming-books-es.md [hunk 2]
```diff
@@ -252,9 +245,6 @@
 
 #### React
 
-* [Desarrollo de Aplicaciones Web con React.js y Redux.js](https://leanpub.com/react-redux) - Sergio Xalambrí (PDF, EPUB, MOBI; Leanpub cuenta requerida)
```
Comment (davorpa) [inline books/free-programming-books-es.md:255]: It's free to read online :worried:

File: books/free-programming-books-id.md (hunk 1 above)
Comment (eshellman) [reply books/free-programming-books-id.md:185]: Didn't want a typescript heading without entries, but it seems other lists are done the way you suggest

File: books/free-programming-books-es.md (hunk 2 above)
Comment (eshellman) [reply books/free-programming-books-es.md:255]: I'll write a script to check them all; when free to read HTML we can remove the access note.

File: books/free-programming-books-id.md (hunk 1 above)
Comment (davorpa) [reply books/free-programming-books-id.md:185]: Let's wait for out community preferences :wink:

File: books/free-programming-books-es.md (hunk 2 above)
Comment (davorpa) [reply books/free-programming-books-es.md:255]: > ... when free to read HTML we can remove the access note.

Suggestion updated

File: books/free-programming-books-id.md (hunk 1 above)
Comment (eshellman) [reply books/free-programming-books-id.md]: changed  it to your suggestion

**Review by davorpa — APPROVED** [review]
Now it's right to my oldy glassed eyes 🤣

**Pull request merged**